cypher/cypher-docs/src/docs/dev/ql/constraints/index.asciidoc

```diff
@@ -33,10 +33,9 @@ Unique node property constraints can be created, along with node and relationshi
  ** <<constraints-drop-a-node-key, Drop a Node Key>>
  ** <<constraints-create-a-node-that-complies-with-a-node-key, Create a node that complies with a Node Key>>
  ** <<constraints-create-a-node-that-violates-a-node-key, Create a node that violates a Node Key>>
+ ** <<constraints-removing-a-node-key-constrained-property, Removing a `NODE KEY`-constrained property>>
```

Comment:
How does this differ from `Drop a Node Key` above?

Reply:
OK. I think I found it below. So this is not about removing the NODE KEY, but what happens when you remove a property that is constrained. Makes sense now.

Reply:
👍